skfem/element/element_tri/element_tri_ccr.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+
+from ..element_h1 import ElementH1
+from ...refdom import RefTri
+
+
+class ElementTriCCR(ElementH1):
+
+    nodal_dofs = 1
+    facet_dofs = 1
+    interior_dofs = 1
+    maxdeg = 3
+    dofnames = ['u', 'u', 'u']
+    doflocs = np.array([[0., 0.],
+                        [1., 0.],
+                        [0., 1.],
+                        [.5, 0.],
+                        [.5, .5],
+                        [0., .5],
+                        [1./3, 1./3]])
+
+    refdom = RefTri
+
+    def lbasis(self, X, i):
+        x, y = X
+
+        if i == 0:  # (0., 0.)
+            phi = -(-2*x + y*(3*x - 2) + 1)*(x + y - 1)
+            dphi = np.array([-6*x*y + 4*x - 3*y**2 + 7*y - 3,
+                             -3*x**2 - 6*x*y + 7*x + 4*y - 3])
+        if i == 1:  # (1., 0.)
```

Comment:
should be `elif` ?

Reply:
maybe it doesn't matter though

Reply:
ahh, of course it does. this causes the index error

Reply:
Ah yes, I think it got jumbled somewhere in copying and rearranging the order. Fixed.

Reply:
And I think I messed up the convergence order of `ElementTriCCR` as well.. Hopefully should work now..